code/datums/skills/_skill.dm

```diff
@@ -16,10 +40,35 @@
   * * old_level - Similar to the above, but the level you had before levelling up.
   */
 /datum/skill/proc/level_gained(var/datum/mind/mind, new_level, old_level)//just for announcements (doesn't go off if the xp gain is silent)
-	to_chat(mind.current, "<span class='nicegreen'>I feel like I've become more proficient at [name]!</span>")
-
+	to_chat(mind.current, levelUpMessages[new_level]) //new_level will be a value from 1 to 6, so we get appropriate message from the 6-element levelUpMessages list
+	try_skill_reward(mind, new_level)
 /**
   * level_lost: See level_gained, same idea but fires on skill level-down
   */
 /datum/skill/proc/level_lost(var/datum/mind/mind, new_level, old_level)
-	to_chat(mind.current, "<span class='warning'>I feel like I've become worse at [name]!</span>")
+	to_chat(mind.current, levelDownMessages[old_level]) //old_level will be a value from 1 to 6, so we get appropriate message from the 6-element levelUpMessages list
+
+/**
+  * try_skill_reward: Checks to see if a user is eligable for a tangible reward for reaching a certain skill level
+  *
+  * Currently gives the user a special cloak when they reach a legendary level at any given skill
+  * Arguments:
+  * * mind - The mind that you'll want to send messages and rewards to
+  * * new_level - The current level of the user. Used to check if it meets the requirements for a reward
+  */
+/datum/skill/proc/try_skill_reward(var/datum/mind/mind, new_level)
+	if (new_level != SKILL_LEVEL_LEGENDARY)
+		return
+	if (!ispath(skillcapepath))	
+		to_chat(mind.current, "<span class='nicegreen'>My legendary [name] skill is quite impressive, thoguh it seems the Professional [title] Association doesn't have any status symbols to commemorate my abilities with. I should let Centcom know of this travesty, maybe they can do something about it.</span>")
+		return
+	if (src in mind.skills_leveled_to_legendary)	
+		to_chat(mind.current, "<span class='nicegreen'>I have regained my legendary [name] skill. It seems the Professional [title] Association won't send me another status symbol, however.</span>")
+		return
+	var/obj/structure/closet/supplypod/bluespacepod/pod = new()
+	pod.landingDelay = 150
+	pod.explosionSize = list(0,0,0,0)
+	to_chat(mind.current, "<span class='nicegreen'>My legendary skill has attracted the attention of the Professional [title] Association. It seems they are sending me a status symbol to commemorate my abilities.</span>")
+	var/turf/T = get_turf(mind.current)
```

Comment:
what if you're in a locker or space or something

Reply:
then it appears ontop the locker or in space